cmd/tk/tool.go

```diff
@@ -52,7 +52,7 @@ func jpathCmd() *cobra.Command {
 func importsCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "imports [file]",
-		Short: "list all transitive imports of a file",
+		Short: "import a transitive file",
```

Comment:
Huh, I do think the old text was correct.

This tool goes ahead prints every direct and recursive import of the file to have a flat list of all dependencies.

So after all, it *lists all transitive / recursive imports of a file*

Reply:
Seems that i misunderstand it with `ls` in shell, so the file param is still not optional?

Reply:
No, because it creates a list (an array) of all recursive imports of _that specific_ file

Reply:
so why not named as "tool list"?